lib/features.js

```diff
@@ -0,0 +1,105 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const hash = require('sha1')
```

Comment:
Yes, siphash is one of fastest cryptographically secure hash functions. Rust uses it in its hash table (so does Python 3.4). However, it still is considered slow for simply converting a variable length string into a number. Since we have no worry about collisions, and if we really wanted speed, we could use something like xxHash or the like. We really only care about an even distribution, and that it's fast.